src/applications/edu-benefits/10203/tests/actions/post-911-gib-status.unit.spec.js

```diff
@@ -8,7 +8,7 @@ const GET_REMAINING_ENTITLEMENT_SUCCESS = 'GET_REMAINING_ENTITLEMENT_SUCCESS';
 let oldWindow;
 const setup = () => {
   mockFetch();
-  setFetchJSONResponse(global.fetch.onCall(0), {});
```

Comment:
I deleted this line because it's unnecessary. There's only one test in this file and it also uses `setFetchJSONResponse`.